core/dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -51,6 +51,29 @@
 
 {% endmacro %}
 
+{% macro get_create_index_sql(relation, index_dict) -%}
+  {{ return(adapter.dispatch('get_create_index_sql')(relation, index_dict)) }}
+{% endmacro %}
+
+{% macro default__get_create_index_sql(relation, index_dict) -%}
+  {{ exceptions.raise_not_implemented(
+    'get_create_index_sql macro not implemented for adapter '+adapter.type()) }}
+{% endmacro %}
+
+{% macro create_indexes(relation) -%}
+  {{ adapter.dispatch('create_indexes')(relation) }}
+{%- endmacro %}
+
+{% macro default__create_indexes(relation) -%}
+  {%- set _indexes = config.get('indexes', default=[]) -%}
+
+  {%- call statement('create_indexes') -%}
```

Comment:
I'm not too sure on the purpose of `call statement`. Is it meant to track calls and their results? For what purpose? Should I be calling `call statement` outside the loop to track the whole `create_indexes` call or inside loop with a different name for each index to track the individual calls?